components/clipboard-button/index.js

```diff
@@ -82,9 +82,9 @@ class ClipboardButton extends Component {
 
 		return (
 			<span ref={ this.bindContainer }>
-				<Button { ...buttonProps } className={ classes }>
+				<IconButton { ...buttonProps } icon="admin-links" className={ classes }>
```

Comment:
Should this have been applied to ClipboardButton broadly, or is it specific to PostPermalink ?

It's affecting other usage, such as the ClipboardButtons shown in the editor crash prompt:

![image](https://user-images.githubusercontent.com/1779930/39492126-94fd9f3a-4d5c-11e8-8577-44c7cedac2a7.png)

Reply:
Oh I missed that sorry :(